runtime/doc/if_lua.txt

```diff
@@ -9,7 +9,109 @@ Lua Interface to Nvim					*lua* *Lua*
                                       Type <M-]> to see the table of contents.
 
 ==============================================================================
-1. Commands						*lua-commands*
+1. Importing modules					*lua-require*
+
+Neovim lua interface automatically adjusts `package.path` and `package.cpath` 
+according to effective &runtimepath value.  Adjustment happens after each time 
+'runtimepath' is changed, `package.path` and `package.cpath` are adjusted by 
+prepending directories from 'runtimepath' each suffixed by `/lua` and 
+`?`-containing suffixes from `package.path` and `package.cpath`. I.e. when 
+'runtimepath' option contains `/foo` and `package.path` contains only 
+`./?.lua;./a?d/j/g.nlua;/bar/?.lua` the resulting `package.path` after 
+adjustments will look like this: >
+
+	/foo/lua/?.lua;/foo/lua/a?d/j/g.nlua;./?.lua;./a?d/j/g.nlua;/bar/?.lua
+
+Note that code have taken everything starting from the last path component 
+from existing paths containing a question mark as a `?`-containing suffix, but 
+only applied unique suffixes.
+
+Note 2: even though adjustments happens automatically Neovim does not track 
```

Comment:
Does this paragraph say that if you adjust `package.path` and `package.cpath` from within a Lua script run by Neovim, the result will be unreliable because the next `runtimepath` change will reset the package path?